web/docs/guides/auth/auth-email.mdx

```diff
@@ -87,11 +87,10 @@ async function signInWithEmail() {
 <TabItem value="dart">
 
 ```dart
-function SignInWithEmail() {
-  await supabase.auth.signIn(
-    email: 'example@email.com',
-    password: 'example-password'
-  );
+Future<void> signInWithEmail() async {
+  await supabase.auth
+      .signIn(email: 'example@email.com', 
+      password: 'example-password');
```

Comment:
Thank you for the fix, but could you make this look a little bit nicer? Maybe something that looks like this?
```suggestion
Future<void> signInWithEmail() async {
  await supabase.auth.signIn(
    email: 'example@email.com', 
    password: 'example-password'
  );
```